### PR TITLE
add node.js websockets reference docs

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: src/pages/reference/nodejs/v0/websocket/websocket-send.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    connectionStream.on('end', res)
+  })
+
+  connectionStream.on('data', async ({ content }) => {
+    // Send message to a connection
+    await socket.send(content.connectionId, data)
+  })
+
+  await streamEnd
+}
```
Comment (tjholm) [inline src/pages/reference/nodejs/v0/websocket/websocket-send.mdx]: not javascript, either need to update the annotation or remove the data types.

File: src/pages/reference/nodejs/v0/websocket/websocket-close.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const closeConnection = async (connectionId) => {
+  await socket.close(connectionId)
+}
+```
+
+## Parameters
+
+<Properties>
+  <Property name="connectionId" required type="string">
+    The ID of the connection which should be closed.
```
Comment (davemooreuws) [inline src/pages/reference/nodejs/v0/websocket/websocket-close.mdx:27]:
```suggestion
    The ID of the connection to be closed.
```

File: src/pages/reference/nodejs/v0/websocket/websocket-on.mdx
Comment (davemooreuws) [inline src/pages/reference/nodejs/v0/websocket/websocket-on.mdx:1]: Looks good. Might be worth adding that query parameters can only be accessed on connect event.

File: src/pages/reference/nodejs/v0/websocket/websocket-send.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+/**
+ * Send a message to a connected client.
+ *
+ * @param connectionId the client received when connecting to the websocket
+ * @param message to send to the client
+ */
+const sendMessage = async (connectionId, message) => {
+  const env = new TextEncoder()
+  await socket.send(connectionId, enc.encode(message))
```
Comment (davemooreuws) [inline src/pages/reference/nodejs/v0/websocket/websocket-send.mdx:21]: Do we need the enc.encode? I thought alot of this was handled internally

File: src/pages/reference/nodejs/v0/websocket/websocket-send.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+socket.on('disconnect', async (ctx) => {
+  await connections.doc(ctx.req.connectionId).delete()
+})
+
+const broadcast = async (data: string | Uint8Array) => {
+  const connectionStream = connections.query().stream()
+
+  const streamEnd =
+    new Promise() <
```
Comment (davemooreuws) [inline src/pages/reference/nodejs/v0/websocket/websocket-send.mdx:74]: Weird formatting

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Looks pretty good! Just some suggestions and questions.

**Review by HomelessDinosaur — DISMISSED** [review]

**Review by tjholm — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**